nightingale_perception/launch/pt_cloud_process.launch

```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0"?>
+<launch>
+  <!-- launch file for point cloud laser scan conversion and filtering of robot body -->
+  <arg name="filtered_points_topic" default="/camera/depth/color/points_body_filtered"/>
+
+  <!-- Point cloud body filter node -->
+  <node name="pointcloud_filter" pkg="sensor_filters" type="pointcloud2_filter_chain" output="screen">
+      <rosparam command="load" file="$(find nightingale_perception)/config/movo_body_filter.yaml" />
+  
+      <remap from="~input" to="/camera/depth/color/points" />
+      <remap from="~output" to="$(arg filtered_points_topic)" />
+  </node>
+
+  <!-- Point cloud to laser scan conversion -->
+  <node pkg="pointcloud_to_laserscan" type="pointcloud_to_laserscan_node" name="pointcloud_to_laserscan_node" output="screen">
+      <remap from="/cloud_in" to="$(arg filtered_points_topic)"/>
+      <!--remap from="/cloud_in" to="/camera/depth/color/points"/-->
+      <remap from="/scan" to="/movo/camera_scan"/>
+
+      <rosparam>
+          target_frame: "front_laser_link" #front laser link chosen for 
```

Comment:
After reviewing the ROS bag data taken before, the laser parameters in the real front laser are slightly off. May need to add an if statement here to check for "MOVO_SIM=true" to use different params.
![real_front_laser_params](https://user-images.githubusercontent.com/42899188/211237710-b08a439d-cde0-45b9-8f49-df02ca743b96.png)

Reply:
Testing in real world with current params seems fine performance